src/common.ts

```diff
@@ -112,6 +124,10 @@ export abstract class Kommand extends Command {
       else {
         await this.runSafe()
       }
+
+      try {
+        await this.analytics.track(kommand.id, this.config.name, this.config.version);
```

Comment:
This should not be awaited to not slow not block the command execution

Reply:
I found a way using Promise.all for both command and analytics